content/id/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -0,0 +1,456 @@
+
+Versi *beta*, dimana mendukung untuk *scaling* merdasarkan memori dan metrik khusus,
+dapat ditemukan pada `autoscaling/v2beta2`. *Field* yang baru diperkenalkan pada
+`autoscaling/v2beta2` adalah *preserved* sebagai anotasi ketika menggunakan `autoscaling/v1`. 
+
+Ketika kamu membuat sebuah HorizontalPodAutoscaler, pastikan nama yang ditentukan adalah valid
+[nama subdomain DNS](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names).
+Untuk lebih detail tentang objek API ini dapat ditemukan di
+[Objek HorizontalPodAutoscaler](https://git.k8s.io/community/contributors/design-proposals/autoscaling/horizontal-pod-autoscaler.md#horizontalpodautoscaler-object).
+
+## Dukungan untuk HorizontalPodAutoscaler pada kubectl
+
+Seperti API *resource* lainnya, HorizontalPodAutoscaler didukung dengan standar oleh `kubectl`.
+Kita dapat membuat *autoscaler* yang baru dengan menggunakan perintah `kubectl create`.
+Kita dapat melihat daftar *autoscaler* dengan perintah `kubectl get hpa` dan melihat deskripsi
+detailnnya dengan perintah `kubectl describe hpa`. Akhirnya, kita dapat menghapus *autoscaler*
+meggunakan perintah `kubectl delete hpa`. 
+
+Sebagai tambahan, terdapat sebuah perintah kusus `kubectl autoscaler` untuk mempermudah pembuatan
+HorizontalPodAutoscaler. Sebagai contoh, mengeksekusi
+`kubectl autoscaler rs foo --min=2 --max=5 --cpu-percent=80` akan membuat sebuah *autoscaler* untuk
+ReplicaSet *foo*, dengan target pengguaan CPU `80%` dan jumlah replika antara 2 sampai dengan 5.
+Dokumentasi lebih detail tentang `kubectl autoscaler` dapat ditemukan di 
+[sini](/docs/reference/generated/kubectl/kubectl-commands/#autoscale).
+
+## Autoscaling ketika Rolling Update
+
+Saat ini, dimungkinkan untuk melakukan *rolling update* menggunakan objek Deployment, dimana 
+mengatur ReplicaSet untuk kamu. HorizontalPodAutoscaler hanya mendukung pendekatan terakhir:
+HorizontalPodAutoscaler terikat dengan objek Deployment, yang mengatur besar dari objek Deployment,
+dan Deployment bertugas untuk mengatur besar dari ReplicaSet.
+
+HorizontalPodAutoscaler tidak bekerja dengan *rolling update* yang menggunakan manipulasi
+pada kontroler replikasi secara langsung, dengan kata lain kamu tidak bisa mengikat
+HorizontalPodAutoscaler dengan kontroler replikasi dan melakukan *rolling update*.
+Alasan HorizontalPodAutoscaler tidak bekerja ketika *rolling update* membuat kontroller
+replikasi yang baru adalah HorizontalPodAutoscaler tidak akan terikat dengan kontroler
+replikasi yang baru tersebut.
+
+## Dukungan untuk *Cooldown* / Penundaan
+
+Ketika mengolah *scale* dari sebuah replika grup menggunakan HorizonalPodAutoscaler,
+jumlah replika dimungkinkan tetap berubah secara sering disebabkan oleh perubahan dinamis
+dari metrik yang dievaluasi. Hal ini sering disebut dengan *thrashing*. 
+
+Mulai dari versi 1.6, operator klaster dapat mengatasi masalah ini dengan mengatur
+setingan HorizontalPodAutoscaler global sebagai *flag* `kube-controller-manager.
```

Comment:
setingan is not indonesian word
also missing backtick for kube-controller-manager
```suggestion
konfigurasi HorizontalPodAutoscaler global sebagai *flag* `kube-controller-manager`.
```